app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -364,11 +362,10 @@ public Chapter getSelectedChapter() {
             return mSelectedChapter;
         }
 
-        public void setSelectedChapter(Chapter chapter) {
+        public void setSelectedChapter(@NonNull Chapter chapter) {

```

Comment:
Yep, actually I didn't change anything. I've just put NonNull, becuase it shouldn't be null. 
Actually we can remove that null check as well, but I don't want to mess with analytics code.